src/scripts/main.js

```diff
@@ -1 +1,82 @@
 'use strict';
+
+const body = document.querySelector('body');
+
+const showMessage = (message, statusNotification) => {
+  const notification = document.createElement('div');
+
+  notification.setAttribute('data-qa', 'notification');
+
+  notification.classList.add(
+    statusNotification === 'success' ? 'success' : 'error',
+  );
```

Comment:
```suggestion
  notification.classList.add(statusNotification);
```
we can simplify that